lib/mix/test/mix/tasks/compile.erlang_test.exs

```diff
@@ -174,4 +174,18 @@ defmodule Mix.Tasks.Compile.ErlangTest do
       assert output == ""
     end)
   end
+
+  @tag erlc_options: [{:warnings_as_errors, true}]
+  test "adds :debug_info to erlc_options by default" do
+    in_fixture("compile_erlang", fn ->
+      Mix.Tasks.Compile.Erlang.run([])
+
+      binary = File.read!("_build/dev/lib/sample/ebin/b.beam")
+
+      {:ok, {module, [debug_info: {:debug_info_v1, backend, data}]}} =
+        :beam_lib.chunks(binary, [:debug_info])
+
+      assert backend.debug_info(:elixir_v1, module, data, []) != {:error, :missing}
```

Comment:
I think this should assert with `:erlang_v1` - after all it's compiling erlang modules, so it's kind of expected it won't be able to return elixir debug info.
```suggestion
      assert backend.debug_info(:erlang_v1, module, data, []) != {:error, :missing}
```

Reply:
Alrighty!

Reply:
I wouldn't even assert this. Asserting it returns a debug info chunk is probably good enough!

Reply:
You mean something like this?

```elixir 
{:ok, {_, [debug_info: {:debug_info_v1, _, {info, _}}]}} = :beam_lib.chunks(binary, [:debug_info])
assert info != :none
```

Reply:
I've checked and the only difference is that it returns `:none` for that part when debug info is disabled, instead of the list of attributes.

Reply:
I have started with that actually, just changed to use `backend.debug_info` because I was afraid that maybe the format could change in the future and then the test could fail. Very unlikely, but who knows 🤷‍♂